subprojects/fwupd-efi.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory = fwupd-efi
+url = https://github.com/fwupd/fwupd-efi
+#TODO: tag 1.0 before merging
```

Comment:
(Need to make sure we don't forget to do this when everything else looks good)

Reply:
If you want me to do a 1.0 tomorrow of fwupd-efi let me know. Then I think we're nearly there with this PR.

Reply:
We're getting really close with it.

Remaining things in my mind:
[ ] Take one more pass at documentation both here and there, make sure it all flows well and explains things
[ ] Pull artifacts out of CI on tagged releases.
[ ] (Maybe) Export the version somewhow so fwupd can use that instead of `fs` module
[ ] (Maybe) Re-spin 1.5.x PR to use that exported version to make a call of building included EFI binary instead of meson option - IE if the PC file is present, then don't build included binary, if it's not then do it.
[ ] Finish Debian packaging for fwupd-efi
[ ] PR's for 1.4.x, 1.3.x, 1.2.x, 1.1.x matching the 1.5.x one.

Reply:
This looks good from my perspective.  The only open question is if whether or not binaries for rpm/deb should be artifacted from CI, and I keep oscillating.  They're not very valuable unsigned.

Reply:
>  They're not very valuable unsigned

My gut opinion is that unsigned-but-latest-version is more valuable than signed-and-old-version, but that's only because I've seen the stats on how many people run SB=enabled in the real world :(

Reply:
But so the thought is that if someone is running a super old RHEL/Ubuntu/Debian they can grab the rpm/deb from Github releases and install the unsigned one in a pinch?  Seems conceptually OK at first until you remember that if they install it on the same system that has an older fwupd userspace that didn't already decouple the binary you can't actually install it directly  because of the conflict...

Reply:
For RHEL 8 and 9 it'll all be one package (too late to split), so that's 10 years of bundling....

Reply:
Did you push tags for 1.0 on fwupd-efi?  I didn't see 'em.

Reply:
mea culpa, done

Reply:
OK then we just need to depend on that tag and I guess squash this to merge.